strapi/strapi
# Fix Can add two attributes with the same name and save

### Why is it needed?

- CTB allows adding an attribute with the same name causing issues when saving

### How to test it?

- Try to add an attribute to a contentType with an attribute name that already exist it should show and error like this
<img width="824" alt="image" src="https://github.com/strapi/strapi/assets/37274596/b23be454-59e1-4e01-8481-fd874313b14a">


### Related issue(s)/PR(s)

fix #20364
